deployment.Rmd

```diff
@@ -58,6 +58,22 @@ library(sparklyr)
 sc <- spark_connect(master = "yarn-client")
 ```
 
+or for more customized configuration:
+
+```{r}
+library(sparklyr)
+sc <- spark_connect(master="yarn",
+                    config = list(
```

Comment:
Ok thanks. Now I see that the deplyoment page is full, but I didn't understand the motivation for config files :)